images/spire/config/main.tf

```
@@ -0,0 +1,43 @@
+variable "name" {
+  description = "Package name (e.g. cainjector, acmeresolver, controller, webhook)"
+}
+
+variable "suffix" {
+  description = "Package name suffix (e.g. version stream)"
+  default     = ""
+}
+
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+
+}
+
+# agent runs as root while others as non-root, this variable is for that
+variable "run-as" {
+  description = "The user with which this should run as"
+  default     = 65532
+}
+
+# agent and server require `run` while oidc-discovery-provider doesn't, this variable tries to handle that
+variable "command" {
+  description = "The user with which this should run as"
```

Seems like this was copypasted from the previous variable!

```suggestion
  description = "The command to be used as the entrypoint"
```

---

Thanks @stormqueen1990 , just fixed it!